src/plugins/controls/public/control_group/control_group.scss

```diff
@@ -123,12 +123,6 @@ $controlMinWidth: $euiSize * 14;
   &--small {
     width: $smallControl;
     min-width: $smallControl;
-
-    &:not(.controlFrameWrapper--grow) {
-      .controlFrame__labelToolTip {
-        max-width: 20%;
-      }
-    }
```

Comment:
Now that the popover can extend past the input for range sliders, I don't think this specific styling is necessary anymore.